its/src/test/java/com/sonar/it/scanner/msbuild/utils/AnalysisContext.java

```diff
@@ -76,8 +78,20 @@ public AnalysisContext setEnvironmentVariable(String name, String value) {
   }
 
   public AnalysisResult runAnalysis() {
+    var result = runAnalysisInternal();
+    assumeTrue(result.isSuccess(), "Analysis END step failed.");
```

Comment:
Are you sure about `assume` here? Shouldn't this be `assert`?
https://junit.org/junit4/javadoc/latest/org/junit/Assume.html

Reply:
🤦